cmd/whereabouts.go

```diff
@@ -103,8 +108,8 @@ func cmdAdd(args *skel.CmdArgs, client *kubernetes.KubernetesIPAM, cniVersion st
 	return cnitypes.PrintResult(result, cniVersion)
 }
 
-func cmdDel(args *skel.CmdArgs, client *kubernetes.KubernetesIPAM) error {
-	logging.Debugf("Beginning delete for ContainerID: %v", args.ContainerID)
+func cmdDel(client *kubernetes.KubernetesIPAM) error {
+	logging.Debugf("Beginning IPAM for podRef: %q - ifName: %q", client.Config.GetPodRef(), client.IfName)
```

Comment:
+1 nice to see it's on a CNI DEL

Reply:
Actually, I forgot to delete those logs (in cmdAdd and cmdDel) since I'm adding logs now 1 level above (cmdAddFunc and cmdDelFunc). Will delete them.